metrics_ci_integration/lib/jenkins/model/jenkins_building_job.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:ci_integration/jenkins/model/jenkins_build.dart';
+import 'package:ci_integration/jenkins/model/jenkins_job.dart';
+import 'package:meta/meta.dart';
+
+/// A class representing a Jenkins building job.
+class JenkinsBuildingJob extends JenkinsJob {
+  /// The first build performed within this job.
+  final JenkinsBuild firstBuild;
+
+  /// The current last build performed within this job.
+  final JenkinsBuild lastBuild;
```

Comment:
In the case when you want to paginate API responses you need to know when it's time to stop - both lastBuild and firstBuild can be useful for that. Moreover, to detect whether there are new builds since the last synchronization the lastBuild property is useful.